include/EditView/Panels.js

```diff
@@ -10,7 +10,7 @@
  * OF NON INFRINGEMENT OF THIRD PARTY RIGHTS.
  *
  * This program is distributed in the hope that it will be useful, but WITHOUT
- * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * ANY WARRANTY; without even the mplied warranty of MERCHANTABILITY or FITNESS
```

Comment:
Dear Ashley


I'm so sorry for this,

I will fix it as soon as I will start work tomorrow


Jerzy

________________________________
From: Ashley Nicolson <notifications@github.com>
Sent: 13 July 2016 16:24:58
To: salesagility/SuiteCRM
Cc: Jerzy Brzezinski; Comment
Subject: Re: [salesagility/SuiteCRM] Adding Events Delegates Filter via Subpanel (#1709)


In include/EditView/Panels.js<https://github.com/salesagility/SuiteCRM/pull/1709#discussion_r70647540>:

> @@ -10,7 +10,7 @@
>   * OF NON INFRINGEMENT OF THIRD PARTY RIGHTS.
>   *
>   * This program is distributed in the hope that it will be useful, but WITHOUT
> - * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
> + * ANY WARRANTY; without even the mplied warranty of MERCHANTABILITY or FITNESS


Hi Jerzy - https://suitecrm.com/wiki/index.php/Coding_Standards#License

No, this typo was added by the commit - there is no typo on the Coding Standards. Update this.

-
You are receiving this because you commented.
Reply to this email directly, view it on GitHub<https://github.com/salesagility/SuiteCRM/pull/1709/files/7f0ea92784711cdc5f696cce53defbe1c3832e0c#r70647540>, or mute the thread<https://github.com/notifications/unsubscribe/AKP8DC7BWRcpSf_IWEmLs2IBD4-yU48Wks5qVQNKgaJpZM4JF9xd>.